wordpress-mobile/WordPress-iOS
# Implement content extension support for rich Comment notifications

## Description 
Fixes #9845, which call for rendering a richer version of our Comment notifications.

## To Test
- Verify that the branch builds & existing tests pass.
- While debugging the content extension, manually push a payload with a valid `note_id` value. Instructions for this can be found here: #9924 
- Force-touch on the _Short Look_ to visually inspect the _Long Look_.

## Expected Results
For the given `note_id`, the _Long Look_ should show a view that includes the entire content of that notification. The following is an example citing the [Gettysburg Address](https://en.wikipedia.org/wiki/Gettysburg_Address). It spans two screenshots because the content is scrollable.
![9845a](https://user-images.githubusercontent.com/221062/44547952-8b491b80-a6d1-11e8-9dfa-bdfe07ab87ed.png)
![9845b](https://user-images.githubusercontent.com/221062/44547953-8b491b80-a6d1-11e8-9a19-ecc535789c92.png)
